digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/dao/queries.scala

```diff
@@ -64,6 +64,8 @@ object Queries {
 
   def nextViitePrimaryKeyId = sql"select viite_general_seq.nextval from dual"
 
+  def nextCommonHistoryValue = sql"select common_history_seq.nextval from dual"
```

Comment:
so deprecating the older sequence must also change this code to ..common_history.nextval..

Reply:
We'll use this one and remove the new one.